packages/app-builder/src/components/Scenario/AstBuilder/AstBuilderNode/Operand/OperandInfos.tsx

```diff
@@ -9,99 +9,90 @@ import {
   getDataTypeTKey,
   getOperatorTypeIcon,
   getOperatorTypeTKey,
-} from './OperandEditor/OperandOption/Option';
+} from './utils';
 
 const MAX_ENUM_VALUES = 50;
 
-interface OperandTooltipProps {
+interface OperandInfosProps {
   children: React.ReactNode;
-  content: React.ReactNode;
-  side?: Tooltip.TooltipContentProps['side'];
-  align?: Tooltip.TooltipContentProps['align'];
-  sideOffset?: Tooltip.TooltipContentProps['sideOffset'];
-  alignOffset?: Tooltip.TooltipContentProps['alignOffset'];
+  className?: string;
+  gutter?: number;
+  shift?: number;
 }
 
-export function OperandTooltip({
+export function OperandInfos({
   children,
-  content,
-  side = 'right',
-  align = 'start',
-  sideOffset,
-  alignOffset,
-}: OperandTooltipProps) {
+  className,
+  gutter,
+  shift,
+}: OperandInfosProps) {
   return (
-    <Tooltip.Root delayDuration={0}>
-      <Tooltip.Trigger tabIndex={-1} asChild>
-        <span>{children}</span>
-      </Tooltip.Trigger>
-      <Tooltip.Portal>
-        <Tooltip.Content
-          side={side}
-          align={align}
-          sideOffset={sideOffset}
-          alignOffset={alignOffset}
-          className="bg-grey-00 border-grey-10 flex max-h-[400px] max-w-[300px] overflow-y-auto overflow-x-hidden rounded border shadow-md"
-        >
-          <div className="flex flex-col gap-2 p-4">{content}</div>
-        </Tooltip.Content>
-      </Tooltip.Portal>
-    </Tooltip.Root>
+    <Ariakit.HovercardProvider
```

Comment:
I needed to change to a new comp to allow "tooltip inside tooltip" (trying to display a tooltip inside a tooltip popover close the tooltip)